tutorial.md

```diff
@@ -69,6 +27,26 @@ to submit your FL job.
 `manager.py` will automatically launch the `aggregator.py` and `executor.py` to start the FL simulation.
 You can either choose to evaluate your large-scale FL experiment over a GPU cluster or test your code on your local machine.
  
+ 
+ ### Test on your local machine by submitting config
+ 
+It is much easy and convenient to first test your code without a GPU cluster. 
+First add an argument `- use_cuda:  False` under `job_conf` in your configuration file `evals/configs/femnist/conf.yml` to indicate you are training without using GPU.
+
+Set `ps_ip` and `worker_ips` to be `10.0.0.1` and `10.0.0.1:[x]` by default, where x represent how many executors you want to run on your local machine.
```

Comment:
10.0.0.1 addresses don't work on Mac machines. Did you try running this instruction on your machine at all?

Reply:
try `localhost` instead of 10.* or 192.* addresses.